bedrock/firefox/tests/test_views.py

```diff
@@ -313,11 +313,20 @@ def setUp(self):
         self.geo_mock = patcher.start()
         self.addCleanup(patcher.stop)
 
-    def _request(self, country):
+    def _request(self, country, locale='de'):
         self.geo_mock.return_value = country
         request = RequestFactory().get('/firefox/os/')
+        request.locale = locale
         return views.firefox_os_geo_redirect(request)
 
+    def test_en_US_always_2_dot_5(self):
```

Comment:
With the waffle switch, we'll need the following decorator above this line:
```python
@override_switch('firefox-os-2.5', active=True)
```

We'll also need to add `from waffle.testutils import override_switch` near the top of the file-- see http://waffle.readthedocs.org/en/stable/testing/automated.html for more details.